crates/core_arch/src/macros.rs

```diff
@@ -1,5 +1,22 @@
 //! Utility macros.
 
+// Helper struct used to trigger const eval errors when a const generic immediate value is
+// out of range.
+#[doc(hidden)]
```

Comment:
`#[doc(hidden)]` is not needed when this is already private to the crate.

Reply:
Right, I've removed it now